### DISCUSSION TITLE
Support for Debian Bookworm

### DISCUSSION THREAD

**Discussion by bforpc** [snipe/snipe-it#14311] [General]
Hi Team,

we have rules, that we cannot install older Debian OS then bookworm.
Is a release date for this os upcomming?

Regards

Bfo

**Comment by bforpc** [comment]
Nobody knows that, or has an idea?

**Comment by btravouillon** [comment]
We are in the same place. Debian 11 EOL is scheduled for July 2024 and we are already planning to upgrade to Debian 12.

From recent change https://github.com/snipe/snipe-it/commit/43c8e98ec7c74f6793d94319dd3f93a4917c6c10 and less recent change https://github.com/snipe/snipe-it/pull/12964, it looks like Snipe-IT 6.3.1 does not support PHP 8.2 yet, which is [the default PHP release in Debian 12 Bookworm](https://packages.debian.org/bookworm/php).

**Reply by btravouillon** [reply]
Also, see https://github.com/snipe/snipe-it/issues/14305

**Reply by bforpc** [reply]
Hi btravouillon,

thank you for this info. So is there a workaround for now?
Maybe toi install php 8.1 also?

Bfo

**Reply by btravouillon** [reply]
For now we will keep Debian 11 and upgrade to the latest 6.3.1. Looks like [v7 is around the corner](https://grokstar.dev/news/snipeit-releases/2023/11/snipe-it-v7-roadmap/) anyway.

**Reply by bforpc** [reply]
Hi,
OK, sounds good and the v7 plan looks also well. I have at maximum 5 weeks to present a solution... its a short time.
I will see.
Thank you for your Infos.

Bfo

**Reply by bforpc** [reply]
If it runs with php 8.1, then yes, i can install debian 12 and add php 8.1 to it.

Bfo

**Reply by mauro-miatello** [reply]
honestly I don't remember if 8.1 is supported, in any case https://packages.sury.org/php/ contains also 8.0

**Comment by mauro-miatello** [comment]
can you install php 8.1 from another repository?
https://computingforgeeks.com/how-to-install-php-on-debian-linux-2/
